test/unittests/execute_call_test.cpp

```diff
@@ -605,8 +605,7 @@ TEST(execute_call, call_indirect_infinite_recursion)
     EXPECT_TRUE(execute(module, 0, {}).trapped);
 }
 
-constexpr int MaxDepth = 2048;
-static_assert(MaxDepth == CallStackLimit);
```

Comment:
The point of this duplicated declaration was that it would be detected in the tests if we adjust this limit.